kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -310,7 +314,9 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
 
         // update active channel count stats
         RequestStats.ACTIVE_CHANNEL_COUNT_INSTANCE.decrementAndGet();
-        log.info("channel inactive {}", ctx.channel());
+        if (log.isDebugEnabled()) {
+            log.debug("channel inactive {}", ctx.channel());
+        }
```

Comment:
The same as previous comment. And since `close()` is only called in `channelInactive`, I think we can merge these two logs into one.

Reply:
Good suggestion. logs merged into `close()` method.